lib/watir-webdriver/browser.rb

```diff
@@ -346,6 +346,10 @@ def assert_exists
         driver.switch_to.default_content
         true
       end
+    rescue Selenium::WebDriver::Error::NoSuchWindowError
```

Comment:
The problem is that sometimes Edge just loses the default context and throws this error, even when the user is doing normal things. This happens several times running our specs if this isn't rescued.